app/models/package.scala

```diff
@@ -208,4 +208,20 @@ package object models {
       postalCode = address.postcode
     )
   }
+
+  implicit class RichPreviousDocument(previousDocument: PreviousDocumentType04) {
+
+    def toPreviousDocumentType06(): PreviousDocumentType06 = {
+      import previousDocument._
+      PreviousDocumentType06(sequenceNumber, typeValue, referenceNumber, complementOfInformation)
+    }
+  }
+
+  implicit class RichPreviousDocuments(previousDocuments: Seq[PreviousDocumentType04]) {
+
+    def toPreviousDocumentType06(): Seq[PreviousDocumentType06] =
+      previousDocuments.map(
+        d => new RichPreviousDocument(d).toPreviousDocumentType06()
+      )
```

Comment:
Very minor but I assume this can be `previousDocuments.map(_.toPreviousDocumentType06())` ?